distribution/changelog.txt

```diff
@@ -18,8 +18,7 @@
 - Fix: [#7327] Abstract scenery and stations don't get fully See-Through when hiding them (original bug).
 - Fix: [#7382] Opening the mini-map reverts the size of the land tool to 1x1, regardless of what was selected before.
 - Fix: [#7402] Edges of neigbouring footpaths stay connected after removing a path that's underneath a ride entrance.
-- Fix: [#7405] Vehicle test now checks for cover only on tiles above
-vehicle.
+- Fix: [#7405] Vehicle test now checks for cover only on tiles above vehicle.
```

Comment:
This should state the bug that has been fixed, rather than the change that has been made.

Reply:
I probably wasn't clear enough, what I mean is an entry like `- Fix: [#7405] Rides can be covered by placing scenery underneath them.`

Reply:
Done.